libr/anal/p/anal_ws.c

```diff
@@ -14,19 +14,17 @@ static ut64 ws_find_label(int l, RIOBind iob) {
 	RIO *io = iob.get_io (&iob);
 	ut64 cur = 0, size = iob.desc_size (io->desc);
 	ut8 buf[128];
-	RAsmOp *aop = R_NEW0 (RAsmOp);
+	RAsmOp aop;;
```

Comment:
superfluous `;`

I like the commit description :)

beauty-fixes for pancake
--